### PR TITLE
Create hds-grandone.mdx

### PR DESCRIPTION
**Pull request by laurakarhu**
Added a page for Grand One competition page

## Related Issue
https://helsinkisolutionoffice.atlassian.net/browse/HDS-1517

Closes #

## How Has This Been Tested?
locally

### CODE REVIEW

File: site/src/docs/about/this-is-hds.mdx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+slug: "/about/this-is-hds"
+title: "Helsinki Design System (HDS) - esittely"
```
Comment (harriplappalainen) [inline site/src/docs/about/this-is-hds.mdx:3]: "esittely" to english = introduction?

**Review by harriplappalainen — COMMENTED** [review]
This is a good start! 👍 I had just one comment.

File: site/src/docs/about/this-is-hds.mdx (hunk 1 above)
Comment (laurakarhu) [reply site/src/docs/about/this-is-hds.mdx:3]: Thanks, I will fix it

**Review by harriplappalainen — APPROVED** [review]
Accepted! Onwards to victory 🏁

**Pull request merged**